### ISSUE TITLE
Feeds with a mix of articles with and without thumbnails are not shown in a clear visual order on a tablet/foldable

### ISSUE THREAD

**Issue by 9aaron** [spacecowboy/Feeder#424]
### Checklist

- [X] I have used the search function for [**OPEN**](https://github.com/spacecowboy/feeder/issues) issues to see if someone else has already submitted the same bug report.
- [X] I have **also** used the search function for [**CLOSED**](https://github.com/spacecowboy/feeder/issues?q=is%3Aissue+is%3Aclosed) issues to see if the problem is already solved and just waiting to be released.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only to occurs with a certain feed, I will include the URL of that feed.

### App version

2.6.33

### Where did you get the app from

Google Play

### Android version

15

### Device model

Google Pixel 9 Pro Fold

### First occurred

Today (although haven't used the app before)

### Steps to reproduce

1. Add a feed with a mix of articles with and without thumbnails, for example https://theconversation.com/au/articles.atom
2. View the feed on a tablet or opened foldable device

### Expected behaviour

I'd expect either
- a list of either article headings and preview text or cards, or
- a grid of cards where each item is the same height

### Current behaviour

The feed is shown as a grid of cards but articles without thumbnails are shorter than those with thumbnails. And there is no placeholder empty image or padding on the latter to make the grid more regular and easy to see the order of cards.
![Screenshot_20241118-200606](https://github.com/user-attachments/assets/61f5e99b-b2cc-46e3-ac92-4d8d6ab82643)
![Screenshot_20241118-200546](https://github.com/user-attachments/assets/0cbf4d3c-9338-4ac7-b16c-c392c75b4c99)

**Comment by spacecowboy** [comment]
You can get closer to what you want by only showing titles in items. This is available in the settings

**Comment by 9aaron** [comment]
Yes, only showing titles helps with order but I prefer to see thumbnails for a visual cue/preview of what the article is about.

Personally, I would prefer a uniform grid even if this needs padding/placeholder thumbnails (perhaps of the feed logo), not sure what others think.

It would even be an improvement to have a setting to display date and time on all articles, not just those published in the past day, to be able to see the order.

Also I would have thought the vast majority of articles (at least for news article-type feeds) would have more than the 'Max lines' setting of lines so each article card would have that number of lines of text.

Thanks for the response anyway, happy for this to be prioritised/deprioritised as you see fit. Just keen to give feedback as I think the app is fantastic and will be my reader moving forward.

**Comment by spacecowboy** [comment]
Try switching to one of the other article styles and see if you like that better?

**Comment by 9aaron** [comment]
What styles are you referring to? Show/hide thumbnails, show only title etc? I explored those prior to raising the issue. The only setting that helps is setting 'Max lines' to a higher number because it reduces the relative difference between the height of an article card with vs without a thumbnail.

**Comment by spacecowboy** [comment]
There is a setting that is called `Article Style` that defaults to `Card`. It is in the `Article list` section.

In particular, try `Compact Card`.

But,... now that I think about it. This setting might be hidden on tables. But I think the `Compact Card` variant could be a better fit for tablets.

I should enable a subset of the styles for tablets instead of just Card.

**Comment by Federico42** [comment]
I can confirm the option for Article Style is hidden/unavailable on tablet (Samsung Tab A9, 8.7", Android 14). 
The feature would be highly appreciated as it is on smartphone. 
Articles' visualization is extremely confusing and distracting, making it almost impossible to use the app (at least for me). 

If this is the case and it's just a matter of hidden feature that shouldn't be too much of a trouble to make it available. Right?

**Comment by spacecowboy** [comment]
Styles will be available on tablets when this https://github.com/spacecowboy/Feeder/pull/440 is released

**Comment by spacecowboy** [comment]
Nothing will "fix" it currently. My point was only that some other style might alleviate the issue due to being more consistent in size. Especially `Compact Card`

**Comment by spacecowboy** [comment]
https://github.com/spacecowboy/Feeder/pull/466
